### PR TITLE
Update pe.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR fix Conecta2TV and other peruvian sterams

Edit: Add Telecolor from https://telecolortv.pe/ and VZTV

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: streams/pe.m3u [hunk 1]
```diff
@@ -419,3 +420,7 @@ http://iptv.perueduca.pe:1935/canal1/canal11/playlist.m3u8
 https://stmv1.srvif.com/topagency2/topagency2/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="error",Wtv (La Verdad y Punto) (Chincha) (720p) [Not 24/7]
 https://v4.tustreaming.cl/wtv/index.m3u8
+#EXTINF:-1 tvg-id="Telecolor.pe" status="online",Telecolor (Yurimaguas) (720p) [Not 24/7]
+https://live.obslivestream.com/telecolormux/index.m3u8
+#EXTINF:-1 tvg-id=VZTV.pe"" status="online",VZ Televisión (720p) [Not 24/7]
```
Comment (freearhey) [inline streams/pe.m3u:425]:
```suggestion
#EXTINF:-1 tvg-id="VZTV.pe" status="online",VZ Televisión (720p) [Not 24/7]
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/pe.m3u [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AntaresTelevision.pe" status="online",Antares Televisión (720p) [Not 24/7]
 https://5c3fb01839654.streamlock.net:1963/iptvantares/liveantarestv/playlist.m3u8
-#EXTINF:-1 tvg-id="Asiritv.pe" status="online",AsiriTV (Lima) (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="Asiritv.pe" status="online",AsiriTV (Lima) (720p) [Not 2
```
Comment (LaneSh4d0w) [inline streams/pe.m3u:4]:
```suggestion
#EXTINF:-1 tvg-id="Asiritv.pe" status="online",AsiriTV (Lima) (720p) [Not 24/7]
```

File: streams/pe.m3u [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
+4/7]
```
Comment (LaneSh4d0w) [inline streams/pe.m3u:5]:
```suggestion
```

**Review by BellezaEmporium — APPROVED** [review]

**Comment by sguinetti** [comment]
All fixed.

**Review by freearhey — APPROVED** [review]

**Pull request merged**